modules/overview-of-object-storage-endpoints.adoc

```diff
@@ -0,0 +1,75 @@
+:_module-type: CONCEPT
```

Comment:
This seems like it might be a reference module?

Reply:
It seems like a Hybrid of a concept and a reference module to me.